requirements.txt

```diff
@@ -2,16 +2,19 @@ Django==1.7.4
 Markdown==2.6
 PyMySQL==0.6.3
 coverage==3.7.1
--e git://github.com/PSU-OIT-ARC/django-arcutils.git@83b821849b6459772843be372cba840daa80454a#egg=django_arcutils-master
+-e git://github.com/PSU-OIT-ARC/django-arcutils.git@83b821849b6459772843be372cba840daa80454a#egg=django_arcutils-dev
 django-cloak==0.1.3
--e git://github.com/PSU-OIT-ARC/djangocas.git@8efa963c8eaea9bc1d8f8411b4c83bda4a5b2379#egg=djangocas-master
+-e git://github.com/PSU-OIT-ARC/djangocas.git@8efa963c8eaea9bc1d8f8411b4c83bda4a5b2379#egg=djangocas-origin/HEAD
```

Comment:
You can change this line to just `djangocas` now instead of the full git path. It's on PyPI.